beat/heart.py

```diff
@@ -1840,20 +1885,38 @@ def choose_backend(
 
     pids = ['stored:' + wave for wave in waveforms]
 
+#################################################################################
```

Comment:
I remember you told me sth about that in our chats. What was the issue? Maybe we should expose these offset-factors  to the gf_config somehow? If we cant figure it out we can also split the PR into that of your shape-file parser and that to the gf_config rework.